tests/ebtc/test_timelock_tx_helpers.py

```diff
@@ -0,0 +1,184 @@
+from brownie import chain
+import pytest
+from helpers.constants import EmptyBytes32
+
+
+def test_schedule_permissions(random_safe):
+    random_safe.init_ebtc()
+
+    ## Dummy data
+    target = random_safe.ebtc.active_pool
+    data = target.setFeeBps.encode_input(100)
+    timelock = random_safe.ebtc.lowsec_timelock
+    delay = timelock.getMinDelay()
+
+    with pytest.raises(AssertionError, match="Error: No role"):
+        random_safe.ebtc.schedule_timelock(
+            timelock, target.address, 0, data, EmptyBytes32, EmptyBytes32, delay + 1
+        )
+
+
+def test_schedule_timelock_permissions_on_target(techops):
+    techops.init_ebtc()
+
+    ## Attempt to schedule grant TechOps minting rights by techopss
+    target = techops.ebtc.authority
+    data = target.setUserRole.encode_input(techops.account, 1, True)
```

Comment:
One element that still I think will be great to have perhaps in a readme or somewhere a reference of the numerical values/magical numbers of for example for the roles used in `setUserRole`. i.e: what is 2? Or 1? Admin or mint?.

Perhaps creating an enum class within the ebtcClasss may be helpful in aiding into the name of the roles such as ROLE.ADMIN etc

Reply:
Good call. Will add all configuration of roles and signatures to the repo, primarily for reference, but also in a way in which they can be leveraged for scripts and tests.

Reply:
resolved https://github.com/ebtc-protocol/ebtc-multisig/pull/3/commits/f4f448d10451d3a2ea524233cd8ae229b7dd55d1